src/libs/actions/Device/getDeviceInfo/getOSAndName/index.ts

```diff
@@ -1,12 +1,4 @@
-import {getOSAndName as libGetOSAndName} from 'expensify-common/lib/Device';
-import GetOSAndName from './types';
+// Don't import this file with '* as Device'. It's known to make VSCode IntelliSense crash.
+import {getOSAndName} from 'expensify-common/lib/Device';
```

Comment:
I am working on the PR, but when testing, I can see the command still passes the snake case params, which are coming from the expensify-common here. Is this intentional? I am a bit confused about this?

Reply:
Yes, you are right @mountiny . I think we should leave these ones as they are, to avoid making additional changes to `expensify-common`. I checked the lib and there are a lot of exported properties there that don't follow naming conventions here.

Reply:
Alright, well then there is not much to do here. I am going to let the backend PR go out for review and update here when ready. I think the `app_version` is the only one

Reply:
Ah this is only for web so in native it will be sent differently

Reply:
>Yes, you are right @mountiny . I think we should leave these ones as they are, to avoid making additional changes to expensify-common. I checked the lib and there are a lot of exported properties there that don't follow naming conventions here.

I disagree, I think we should update them all to match our style guide. Is there a reason that can't be done? I understand it's more work, but we should follow our style guide

Reply:
Surely we can do it, my point is: We defined and set these naming conventions for TS files in E/App, so it's the style guide of this project. `expensify-common` is a different library that doesn't even have a defined style guide I think, so change things in a library that is used across different Expensify projects just to satisfy E/App needs looks a bit odd to me 🤔

Reply:
But netherless, in case we would do such changes we shouldn't block this PR because of it, as I understand `expensify-common` is used in several other projects, right?

Reply:
> Surely we can do it, my point is: We defined and set these naming conventions for TS files in E/App, so it's the style guide of this project. `expensify-common` is a different library that doesn't even have a defined style guide I think, so change things in a library that is used across different Expensify projects just to satisfy E/App needs looks a bit odd to me 🤔

I'm pretty sure we use our [public style guide](https://github.com/Expensify/Style-Guide) in all our repos, at least our internal ones. But maybe our App style guide is different. 

> But netherless, in case we would do such changes we shouldn't block this PR because of it, as I understand `expensify-common` is used in several other projects, right?

If there's no updates needed in this code base after updating expensify-common, then we shouldn't block this PR on that update. Otherwise, I think we should update expensify-common first

Reply:
I will update it tomorrow

Reply:
Hmm I don't have access to this repo, but this naming conventions is an ESLint rule that we applied only to TS files in E/App.

> If there's no updates needed in this code base after updating expensify-common, then we shouldn't block this PR on that update. Otherwise, I think we should update expensify-common first

@mountiny @cead22 We would have to change the params that are using snake case. So, do you want to proceed with expensify-common PR first and putting this on HOLD?

cc @kubabutkiewicz Since I will be OOO for the next two weeks.

Reply:
I will make the expensify-common change